[2019-04-03 17:22:43,560] [DEBUG] [D:\zookeeper\zookeeper-3.3.6\gee\my_rf\test_case\test_用户管理_查询用户.py : test_select_all:36 , 当前方法: test_select_all
[2019-04-03 17:22:43,574] [DEBUG] [D:\zookeeper\zookeeper-3.3.6\gee\my_rf\test_case\test_用户管理_查询用户.py : test_select_all:44 , 是测试点"SelectMock"下的用例"查询所有",返回的结果res={'code': '00', 'msg': [{'id': 1, 'uname': 'tom'}, {'id': 2, 'uname': 'jack'}, {'id': 3, 'uname': 'lucy'}]}]
[2019-04-03 17:22:43,583] [DEBUG] [D:\zookeeper\zookeeper-3.3.6\gee\my_rf\test_case\test_用户管理_查询用户.py : test_select_one_exist_first:57 , 是测试点"SelectMock"下的用例"查询一个存在的",返回的结果res={'code': '00', 'msg': {'id': 1, 'uname': 'tom'}}]
[2019-04-03 17:22:43,593] [DEBUG] [D:\zookeeper\zookeeper-3.3.6\gee\my_rf\test_case\test_用户管理_查询用户.py : test_select_one_exist_last:70 , 是测试点"SelectMock"下的用例"查询一个存在的",返回的结果res={'code': '00', 'msg': {'id': 3, 'uname': 'lucy'}}]
[2019-04-03 17:22:43,603] [DEBUG] [D:\zookeeper\zookeeper-3.3.6\gee\my_rf\test_case\test_用户管理_查询用户.py : test_select_one_notExist:83 , 是测试点"SelectMock"下的用例"查询一个不存在的",返回的结果res={'code': '01', 'msg': 'user does not exist'}]
[2019-04-03 17:22:43,610] [DEBUG] [D:\zookeeper\zookeeper-3.3.6\gee\my_rf\test_case\test_计算器_减法.py : test_int_float:69 , 是测试点"MyCacl"下的用例"整数减小数",返回的结果res=7.2]
[2019-04-03 17:22:43,616] [DEBUG] [D:\zookeeper\zookeeper-3.3.6\gee\my_rf\test_case\test_计算器_减法.py : test_int_int:62 , 当前方法: test_int_int
[2019-04-03 17:22:43,621] [DEBUG] [D:\zookeeper\zookeeper-3.3.6\gee\my_rf\test_case\test_计算器_减法.py : test_int_int:63 , 是测试点"MyCacl"下的用例"整数减整数",返回的结果res=0]
[2019-04-03 17:22:43,625] [DEBUG] [D:\zookeeper\zookeeper-3.3.6\gee\my_rf\test_case\test_计算器_减法.py : test_int_letter:84 , 是测试点"MyCacl"下的用例"整数减字母",返回的结果res=类型错误]
[2019-04-03 17:22:43,633] [DEBUG] [D:\zookeeper\zookeeper-3.3.6\gee\my_rf\test_case\test_计算器_减法.py : test_int_negativeNumber:75 , 是测试点"MyCacl"下的用例"整数减负数",返回的结果res=6]
[2019-04-03 17:22:43,633] [DEBUG] [D:\zookeeper\zookeeper-3.3.6\gee\my_rf\test_case\test_计算器_加法.py : test_cacl:55 , 是测试点"MyCacl2"下的用例"整数相加",返回的结果res=3]
[2019-04-03 17:22:43,633] [DEBUG] [D:\zookeeper\zookeeper-3.3.6\gee\my_rf\test_case\test_计算器_加法.py : test_cacl:55 , 是测试点"MyCacl2"下的用例"小数加整数",返回的结果res=3.5]
[2019-04-03 17:22:43,634] [DEBUG] [D:\zookeeper\zookeeper-3.3.6\gee\my_rf\test_case\test_计算器_加法.py : test_cacl:55 , 是测试点"MyCacl2"下的用例"整数加字母",返回的结果res=类型错误]
[2019-04-03 17:22:43,635] [DEBUG] [D:\zookeeper\zookeeper-3.3.6\gee\my_rf\test_case\test_计算器_加法.py : test_cacl:55 , 是测试点"MyCacl2"下的用例"整数加特殊字符",返回的结果res=类型错误]
[2019-04-03 17:22:43,636] [DEBUG] [D:\zookeeper\zookeeper-3.3.6\gee\my_rf\test_case\test_计算器_加法.py : test_cacl:55 , 是测试点"MyCacl2"下的用例"小数相加",返回的结果res=4.1]
[2019-04-03 17:22:43,636] [DEBUG] [D:\zookeeper\zookeeper-3.3.6\gee\my_rf\test_case\test_计算器_加法.py : test_cacl:55 , 是测试点"MyCacl2"下的用例"负数加整数",返回的结果res=1]
[2019-04-03 17:22:43,636] [DEBUG] [D:\zookeeper\zookeeper-3.3.6\gee\my_rf\test_case\test_计算器_加法.py : test_cacl:55 , 是测试点"MyCacl2"下的用例"负数加小数",返回的结果res=0.5]
[2019-04-03 17:22:43,637] [DEBUG] [D:\zookeeper\zookeeper-3.3.6\gee\my_rf\test_case\test_计算器_加法.py : test_cacl:55 , 是测试点"MyCacl2"下的用例"负数相加",返回的结果res=-3]
[2019-04-03 17:22:43,637] [DEBUG] [D:\zookeeper\zookeeper-3.3.6\gee\my_rf\test_case\test_计算器_加法.py : test_cacl:55 , 是测试点"MyCacl2"下的用例"0加整数",返回的结果res=1]
[2019-04-03 17:22:43,637] [DEBUG] [D:\zookeeper\zookeeper-3.3.6\gee\my_rf\test_case\test_计算器_加法.py : test_cacl:55 , 是测试点"MyCacl2"下的用例"0加小数",返回的结果res=0.5]
[2019-04-03 17:22:43,638] [DEBUG] [D:\zookeeper\zookeeper-3.3.6\gee\my_rf\test_case\test_计算器_加法.py : test_cacl:55 , 是测试点"MyCacl2"下的用例"0加负数",返回的结果res=-1]
[2019-04-03 17:22:43,638] [DEBUG] [D:\zookeeper\zookeeper-3.3.6\gee\my_rf\test_case\test_计算器_加法.py : test_cacl:55 , 是测试点"MyCacl2"下的用例"整数加空",返回的结果res=类型错误]
